### PR TITLE
log-conventions

### PR DESCRIPTION
**Pull request by mkawalek**
* 11.8 hull node upgrade
* logs follow new log-convention document

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "cheerio": "^0.22.0",
     "ejs": "^2.5.6",
     "express": "^4.14.0",
-    "hull": "^0.11.4",
+    "hull": "^0.11.8",
```
Comment (michaloo) [inline package.json]: @mkawalek let's not use `^`, but fixed version.

File: server/lib/update-user.js [hunk 2]
```diff
@@ -14,144 +14,136 @@ module.exports = function userUpdate(ctx: Object, messages:Array<Object> = [], {
...
-        client.logger.info("outgoing.user.success", logIdent);
+        asUser.logger.debug("outgoing.user.datanyze.traits.send", { ...payload, userId });
+        asUser.logger.info("outgoing.user.success");
         metric.increment("ship.outgoing.users");
 
         return client.asUser(userId).traits(payload, { source: "datanyze" });
       }, err => {
-        client.logger.debug("fetch.error", { error: err.stack || err, ...logIdent });
-        client.logger.error("outgoing.user.error", { ...logIdent, reason: err });
+        asUser.logger.debug("outgoing.user.fetch.error", { errors: err.stack || err });
```
Comment (michaloo) [inline server/lib/update-user.js]: let's remove that debug, since we have everything in `outgoing.user.error`

File: server/lib/update-user.js [hunk 3]
```diff
@@ -14,144 +14,136 @@ module.exports = function userUpdate(ctx: Object, messages:Array<Object> = [], {
...
           return Promise.resolve();
         }
 
         const technologies = _.map(data.technologies, t => t.name);
         const payload = { ...data, technologies };
         payload.fetched_at = new Date().toISOString();
-        client.logger.debug("datanyze.traits.send", { ...payload, userId });
-        client.logger.info("outgoing.user.success", logIdent);
+        asUser.logger.debug("outgoing.user.datanyze.traits.send", { ...payload, userId });
+        asUser.logger.info("outgoing.user.success");
```
Comment (michaloo) [inline server/lib/update-user.js]: let's add payload to `outgoing.user.success` and remove `outgoing.user.datanyze.traits.send`

File: server/lib/update-user.js [hunk 4]
```diff
@@ -14,144 +14,136 @@ module.exports = function userUpdate(ctx: Object, messages:Array<Object> = [], {
...
                   message,
                   attempt
                 }, {
                   delay: process.env.ADD_DOMAIN_DELAY || 1800000
                 });
-              }, (err) => client.logger.debug("fetch.addDomain.queue.error", { err, ...logIdent }));
+              }, (err) => asUser.logger.debug("fetch.addDomain.queue.error", { errors: err }));
           }
-          client.logger.debug("fetch.addDomain.error", { attempt, domain, error: data.error, ...logIdent });
+          asUser.logger.debug("outgoing.user.fetch.addDomain.error", { attempt, domain, errors: data.error });
```
Comment (michaloo) [inline server/lib/update-user.js]: README.md?

Comment (mkawalek) [inline server/lib/update-user.js]: @michaloo shouldn't we put only info/warn/error logs in README.md ?

Comment (michaloo) [inline server/lib/update-user.js]: @mkawalek good point :) you're right.

**Pull request merged**